cpp/src/flightsql_odbc/flightsql-odbc/build_win32.bat

```diff
@@ -48,5 +48,3 @@ cmake ..^
     -DCMAKE_BUILD_TYPE=release
 
 cmake --build . --parallel 8 --config Release
-
-cd ..
```

Comment:
Is there a way to cd back to upper directory and failing the workflow at the same time?

Reply:
I'm thinking that the build script would be impacted due to this change

Reply:
No problem, I change to capture the result and returned later